### DISCUSSION TITLE
Performance issues

### DISCUSSION THREAD

**Discussion by antmorzam** [maplibre/flutter-maplibre-gl#332] [General]
Could someone with experience in `flutter-maplibre-gl` give us some recommendations for optimal performance and a pleasant user experience?

We have developed a POC using `flutter-maplibre-gl` and we have performance problems in the rendering of the map. 
We have also developed a POC with `maplibre-native` and we have perfectly verified that the performance is good, that's why we doubt the performance we get using `flutter-maplibre-gl`.

**Comment by mariusvn** [comment]
what kind of performance issues do you have ? more like initialization issues or framerate issues ?

**Reply by pabloCactussoft** [reply]
Hi, looks like a little of both. In my case I'm worry about framerate, I was seeing some black spaces on zooming. I implement thunderforest https://www.thunderforest.com/docs/map-tiles-api/

**Comment by antmorzam** [comment]
@mariusvn we finally found the problem. Performance was poor using the simulator. After testing the POC on real devices, we have found that the performance is optimal. Thanks so much
